kernel/main.cpp

```diff
@@ -109,7 +129,26 @@ extern "C" void px_kernel_main(const multiboot_info_t* mb_struct, uint32_t mb_ma
     px_kprintf(DBG_OKAY "Done.\n");
     px_kernel_boot_tone();
     // Keep the kernel alive.
+    px_kprintf("\n");
```

Comment:
Not to be picky, but this code probably shouldn't be in this PR.

Reply:
Removed.